jkube-kit/common/src/test/java/org/eclipse/jkube/kit/common/util/EnvUtilTest.java

```diff
@@ -62,4 +367,10 @@ public void testLoadTimestampShouldLoadFromFile() throws Exception {
         // Then
         assertThat(timestamp, equalTo(new Date(1445385600000L)));
     }
+
+    @Test
+    public  void testIsWindows(){
+        //Then
+        assertEquals(false,isWindows());
```

Comment:
Platform dependent test. Try to make it platform independent and provide two tests, one that verifies it is windows and one that verifies it's not.